lib/adapters/REST/endpoints/resource-type.ts

```diff
@@ -44,3 +58,13 @@ export const getMany: RestEndpoint<'ResourceType', 'getMany'> = (
     `/organizations/${params.organizationId}/app_definitions/${params.appDefinitionId}/resource_provider/resource_types`
   )
 }
+
+export const getForEnvironment: RestEndpoint<'ResourceType', 'getForEnvironment'> = (
```

Comment:
would this be better as `getManyForEnvironment`? definitely not mandating that, just wanted to ask in case that naming helps better convey what this is used for

Reply:
I used the same naming for EnvironmentTemplateInstallation which is also a getMany.